common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -88,7 +88,8 @@ object ActivationMessage extends DefaultJsonProtocol {
  */
 case class CompletionMessage(
     override val transid: TransactionId,
-    response: WhiskActivation)
+    response: WhiskActivation,
+    invoker: String)
```

Comment:
We only know it in the LoadBalancer itself, but this information is not passed to the Actor, that handles the health of the invoker.
That was the initial idea of this field. But your point was right as well.